# the name of the database which will be maintained (see grasp_planning_graspit GraspItDatabaseManager).
database_name: GraspItDatabase

# the name of the service topic to be published for  grasp_planning_graspit_msgs::AddToDatabase.srv
add_to_db_topic: graspit_add_to_database

# the name of the service topic to be published for grasp_planning_graspit_msgs::LoadDatabaseModel.srv
load_model_topic: graspit_load_model

# the name of the service topic to be published for grasp_planning_graspit_msgs::SaveWorld.srv
save_world_topic: graspit_save_world

# the name of the service topic to be published for manipulation_msgs::GraspPlanning.srv
eg_planning_topic: graspit_eg_planning

# the default maximum planning steps to use for the Eigengrasp planner.
default_max_planning_steps: 70000

# the default number of times the planning is repeated (complete planner re-starts).
default_num_repeat_planning: 1

# factor by which all position values are multiplied
# before a grasp result is being saved in a manipulation_msgs/Grasp.msg message.
grasp_msg_position_factor: 0.001

# specifies whether to save the results as inventor files in the folder specified in "results_output_directory".
save_result_files_inventor: true
   
# specifies whether to save the results as graspit world files in the folder specified in "results_output_directory".
save_result_files_graspit: true
